### DISCUSSION TITLE
There is a loud glitchy noise

### DISCUSSION THREAD

**Discussion by vrka367** [namidaco/namida#381] [Q&A]
When on Wired headphones or earphones sometimes the player makes a loud glitchy buzz noise at the beginning (just for a micro second)
This happens when:
- While playing audio again after pausing it
- While turning Equalizer off 
- Sometimes during the beginning (when i play audio for the first time) fresh off the start.

it's getting annoying and started to hurt my ears🫨 some time Hope you fix this 
Yeah and i checked this with another music app to see if the issue is on the earphones or audio file but they were fine but here there was a problem 
The issue still exists in the latest ver4.9 beta
Love the app Thanks a lot for this gorgeous and functional music player

**Comment by MSOB7YY** [comment]
woah sorry for late reply just saw this

is it similar to #470 ?

**Reply by vrka367** [reply]
Yes this thing is bugging bro 😭
Sometimes i forget this and fuk my eardrum Accidentally at full volume 

U have any solution, fix this if it can be fixed 🙏🏻

**Reply by MSOB7YY** [reply]
not really but how about nuking the whole namida equalizer thing and just use external one? like poweramp equalizer or a system one, since they should be always attached with the system audio session

**Reply by vrka367** [reply]
could work but are they similar cause current equalizer is good n won't other uses be pissed?
if u still want to remove it don't remove the 4 controls u have {loudness enhancer, pitch} n nuke it n local one would be enough


check this one out this a cool equalizer options : https://github.com/Moriafly/SaltPlayerSource.git
https://play.google.com/store/apps/details?id=com.salt.music&pcampaignid=web_share
🫡 would request something similar if possible

**Reply by MSOB7YY** [reply]
> won't other users be pissed?

the issue is specific and not everyone face it, the equalizer is a simple one for whoever needs it, but if u really want a good equalizer u shouldnt be using it only

salt player is very good with decoders and audio effects, not namida focus tho

**Reply by vrka367** [reply]
✌🏻mm maybe in future 😁 would love to see it in namida
If everyone is ok with it nuke the Equalizer if not it's ok just cause of few users why remove it others might want it

**Reply by MSOB7YY** [reply]
u misunderstood i didnt mean to remove it hgahgaha i meant that **you** should nuke it (not use it)

**Reply by vrka367** [reply]
Ooh ok 
I'm using local one rn so all good
